### PR TITLE
Add starto

### LINKED ISSUE #6152: strato
**Brand Name:**strato
**Website:**strato.de
**Alexa rank:**14,719
  

**Official resources for icon and color:**https://www.strato.de/_assets/svg/seal/strato_logo_white_xs.svg

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
strato

**Issue:** closes #6152 

**Similarweb rank:** 10,515. See https://www.similarweb.com/website/strato.de

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website header https://www.strato.de
Color: Taken from website header `#ff8800

>**Note**
>The icon is not exactly square but 23.9989x24.0px

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -12219,6 +12219,11 @@
             "hex": "2F2E8B",
             "source": "https://strapi.io/newsroom"
         },
+        {
+            "title": "Strato",
+            "hex": "FF8800",
+            "source": "https://www.strato.de"
```
Comment (adamrusted) [inline _data/simple-icons.json:12225]:
```suggestion
            "source": "https://www.strato.de/press",
            "guidelines": "https://www.strato.de/press"
```

The forbidden line about requiring permission, would you mind reaching out @PratyushJayachandran?

Comment (PratyushJayachandran) [reply _data/simple-icons.json:12225]: Hah okay. I'll do.

Comment (PratyushJayachandran) [reply _data/simple-icons.json:12225]: Awaiting reply from Strato

Comment (PratyushJayachandran) [reply _data/simple-icons.json:12225]: No reply 😴 
Shall we just close this?

Comment (adamrusted) [reply _data/simple-icons.json:12225]: Might be best. Could you also comment on the issue and close it as a won't add?

**Comment by PratyushJayachandran** [comment]
No response from brand upon requesting use of icon. If someone can get the permission feel free to comment for reopening this issue.

**Pull request closed without merging**